reference/release-notes/1.21.2.md

```diff
@@ -0,0 +1,29 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.21.2.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+### Core stack improvements
+
+* WiFi support for AAOS images.<!--AC-2211-->
+
+### Other
+
+* Android security updates for March 2024 (see [Android Security Bulletin - April 2024](https://source.android.com/docs/security/bulletin/2024-04-01) for more information).<!--AC-2391-->
+* The Android WebView has been updated to [123.0.6312.99](https://chromereleases.googleblog.com/2024/04/chrome-for-android-update.html).
+
+### Bug fixes
+
+* [LP 2051220](https://bugs.launchpad.net/anbox-cloud/+bug/2051220) Unable to initialise the Anbox Cloud Appliance when it is behind a HTTP proxy.<!--AC-2306-->
+* [LP 2055497](https://bugs.launchpad.net/anbox-cloud/+bug/2055497) Native crash occurs in the Audio HAL module when booting up the Android system.<!--AC-2330-->
+* [LP 2054443](https://bugs.launchpad.net/anbox-cloud/+bug/2054443) Fix for the session crash when opening an installed app.<!--AC-2297-->
+* Fix for Android SystemUI terminating abruptly with multiple runtime errors.<!--AC-2267-->
```

Comment:
I'd suggest keeping this item out from the release announcement since we didn't technically fix it; rather, we've made adjustments to keep the CI silent when this FATAL exception occurs.

Reply:
@ajanon could you comment on this as well?
Thanks

Reply:
Hum, true, let's remove that one.